dll/win32/comctl32/pager.c

```diff
@@ -1489,7 +1494,16 @@ PAGER_WindowProc (HWND hwnd, UINT uMsg, WPARAM wParam, LPARAM lParam)
 /*      case PGM_GETDROPTARGET: */
 
         case PGM_RECALCSIZE:
+#ifdef __REACTOS__
+            if (!infoPtr->m_bProcessingReCalcSize)
+            {
+                infoPtr->m_bProcessingReCalcSize = TRUE;
+                PostMessageW(hwnd, EM_FMTLINES, wParam, lParam);
+            }
+            return DefWindowProcW(hwnd, uMsg, wParam, lParam);
+#else
             return PAGER_RecalcSize(infoPtr);
```

Comment:
Is this function (`PAGER_RecalcSize`) used also elsewhere in the comctl32 code?
Also, it now looks unnatural that `PAGER_RecalcSize` (which was meant to be the handler of `PGM_RECALCSIZE`) is precisely NOT invoked anymore in that case.

Reply:
Fixed.